pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -518,7 +523,7 @@ func TestPreFilterState(t *testing.T) {
 				DefaultConstraints: tt.defaultConstraints,
 				DefaultingType:     config.ListDefaulting,
 			}
-			p := plugintesting.SetupPluginWithInformers(ctx, t, New, args, cache.NewSnapshot(tt.existingPods, tt.nodes), tt.objs)
+			p := plugintesting.SetupPluginWithInformers(ctx, t, frameworkruntime.FactoryAdapter(plfeature.Features{}, New), args, cache.NewSnapshot(tt.existingPods, tt.nodes), tt.objs)
```

Comment:
You can create a variable that can be used here to make code cleaner. 
You can refer to this https://github.com/kubernetes/kubernetes/blob/45b6af6b71db0c5c61596add8b3f2c67dd071727/pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go#L98